garrysmod/gamemodes/terrortown/gamemode/init.lua

```diff
@@ -91,6 +94,8 @@ CreateConVar("ttt_det_credits_starting", "1")
 CreateConVar("ttt_det_credits_traitorkill", "0")
 CreateConVar("ttt_det_credits_traitordead", "1")
 
+-- Other
+CreateConVar("ttt_scoreboard_sorting", "1", FCVAR_NOTIFY + FCVAR_REPLICATED)

```

Comment:
Good idea; but I believe I need to keep the ConVar as I could not find a shared object needed to host the variable. There may be one though; I'm not very familiar with the codebase.